appium/webdriver/extensions/images_comparison.py

```diff
@@ -147,4 +151,13 @@ def _add_commands(self) -> None:
 
 
 def _adjust_image_payload(payload: Base64Payload) -> str:
-    return payload if isinstance(payload, str) else payload.decode('utf-8')
+    if isinstance(payload, str):
+        return payload
+    try:
+        b64_str = payload.decode('ascii')
```

Comment:
I don't think this is a valid change. The payload is already expected to be b64-encoded, represented as a string or bytes